libkbfs/conflict_resolver.go

```diff
@@ -2873,6 +2895,12 @@ func (cr *ConflictResolver) syncBlocks(ctx context.Context, lState *lockState,
 		}
 	}
 
+	defer func() {
```

Comment:
why is this defer all the way down here? I would have expected it to be right below the `syncTree` call...